test/jaas-test/src/main/resources/suite/suite.conf.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans:beans
+        xmlns="http://www.griddynamics.com/schema/jagger"
+        xmlns:beans="http://www.springframework.org/schema/beans"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation=
+                "
+        http://www.griddynamics.com/schema/jagger
+        http://www.griddynamics.com/schema/jagger.xsd
+        http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans.xsd"
+        >
+
+    <beans:import resource="collectors.conf.xml"/>
+    <beans:import resource="test.description.conf.xml"/>
+
+    <!--Name of workload configuration-->
+    <configuration id="JaaS_TestConfiguration">
+        <latency-percentiles>
+            <percentile>99</percentile>
+        </latency-percentiles>
+
+        <test-suite id="ts_JaaSTestSuit">
+            <test-group id="tg_JaaS_GET_Sessions">
+                <test id="t_JaaS_GET_Sessions_List_SR_eq._1.0" testDescription="td_JaaS_SessionsList">
+                    <load xsi:type="load-user-groups">
+                        <user count="2" startCount="2" startIn="0" startBy="0" life="12s" id="tl_JaaS_sessions" />
```

Comment:
good practice to parametrize life and maxDuration via properties. xml parser can handle it
in this case you can change duration of the test without recompiling